modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/to/Chapters.kt

```diff
@@ -8,23 +8,29 @@ data class Chapters(private val items: List<Chapter> = emptyList()) {
     val size: Int
         get() = items.size
 
-    fun getNextChapter(timeMs: Int): Chapter? {
+    private val selectedItems: List<Chapter>
+        get() = items.filter { it.selected }
+
+    val lastChapter: Chapter
+        get() = items[items.size - 1]
```

Comment:
Shouldn't this be `Chapter?` with `items.getOrNull()`?

Reply:
Yes! updated in https://github.com/Automattic/pocket-casts-android/pull/1850/commits/f8db0177162546e025f46b67832b3d2ab739e3a0